packages/cli/src/commands/init.ts

```diff
@@ -28,137 +129,52 @@ export default class Init extends BaseCommand {
-              answers: Record<string, unknown>
-            ) => {
-              const accountId = await ambientAwsAccountId(answers.region as string)
-              return accountId ? s3DefaultBucketName({ profileAlias, accountId }) : undefined
-            },
-          },
-        ])
-
-        location = `s3://${bucket}?region=${region}`
-      } else if (locationType === 'gs') {
-        // eslint-disable-next-line no-use-before-define
-        const { project, bucket } = await inquirer.prompt<{ project: string; bucket: string }>([
-          {
-            type: 'input',
-            name: 'project',
-            message: 'Google Cloud project',
-            default: driver === 'gce' ? driverFlags['project-id'] : defaultGceProjectId(),
-          },
-          {
-            type: 'input',
-            name: 'bucket',
-            message: 'Bucket name',
-            default: (
-              answers: Record<string, unknown>,
-            ) => gsDefaultBucketName({ profileAlias, project: answers.project as string }),
-          },
-        ])
-
-        location = `gs://${bucket}?project=${project}`
-      } else {
-        location = `local://${profileAlias}`
-      }
-
-      await this.config.runCommand('profile:create', [
-        '--use',
-        '--log-level', this.flags['log-level'] ?? 'info',
-        profileAlias,
-        location,
-        '--driver', driver,
-        ...Object.entries(driverFlags)
-          .filter(excludeDefaultFlags(driverStatic.flags as Record<string, Flag<unknown>>))
-          .map(([key, value]) => {
-            const driverKey = `${driver}-${key}`
-            if (typeof value === 'boolean') {
-              return value ? `--${driverKey}` : `--no-${driverKey}`
-            }
-            if (typeof value === 'string' || typeof value === 'number') {
-              return `--${driverKey}=${value}`
-            }
-            throw new Error(`Cannot convert ${value} of type ${typeof value} to a flag, for flag ${driverKey}`)
-          }),
-      ])
+      return undefined
+    }
+
+    const driver = await chooseDriver()
+    const driverStatic = machineDrivers[driver]
+
+    const driverAnswers = await inquirer.prompt<Record<string, unknown>>(await driverStatic.questions())
+    const driverFlags = await driverStatic.flagsFromAnswers(driverAnswers) as Record<string, unknown>
+
+    const locationType = await chooseLocationType()
+
+    const location = await chooseLocation[locationType]({ profileAlias, driver, driverFlags })
+
+    await this.config.runCommand('profile:create', [
+      '--use',
+      '--log-level', this.flags['log-level'] ?? 'info',
+      profileAlias,
+      location,
+      '--driver', driver,
+      ...formatDriverFlagsToArgs(driver, driverStatic.flags as Record<string, Flag<unknown>>, driverFlags),
+    ])
+
+    ux.info(text.recommendation('Use Livecycle together with Preevy to enable easy sharing and collaboration of your environments!'))
+
+    if (!await confirm({
```

Comment:
I think we should emphasize to `new/existing Livecycle account` so it'll be clear that having an account is not a prerequisite. Alternatively, use something like `register profile on Livcycle`